libs/application/templates/public-debt-payment-plan/src/fields/EmployerInfo/EmployerInfo.tsx

```diff
@@ -1,19 +1,23 @@
-import React from 'react'
+import { PaymentScheduleEmployer } from '@island.is/api/schema'
 import { FieldBaseProps } from '@island.is/application/core'
 import { Box, Text } from '@island.is/island-ui/core'
+import React from 'react'
+import { PaymentPlanExternalData } from '../../lib/dataSchema'
+
+export const EmployerInfo = ({ application }: FieldBaseProps) => {
+  const employerInfo = (application.externalData as PaymentPlanExternalData)
+    .paymentPlanPrerequisites?.data?.employer as PaymentScheduleEmployer
 
-export const EmployerInfo = (props: FieldBaseProps) => {
-  // TODO: Get employer info
-  const employerInfo = {
-    employerSsn: '450199-3389',
-    employerName: 'Bónus ehf.',
+  if (!employerInfo) {
+    // TODO: What should happen?
```

Comment:
Should probably be an error message here that gets displayed to the user in that case.

Reply:
I think this should be a separate task? What happens if there is no employer? Should you still be able to choose that field?